code/api/src/modules/user/tests/userMutations.test.js

```diff
@@ -14,4 +16,54 @@ describe('user mutations', () => {
       })
     )
   })
+
+  beforeAll(async () => {
```

Comment:
@Stearnzy you did a great job figuring out this syntax! Especially this `models.User.create` - with missing Rails magic, having to call on the `models` import definitely didn't come intuitively to me!

Reply:
Thanks!! Took some looking around to find the `models` syntax, but I'm glad it worked too!